### PR TITLE
8260343: Delete obsolete classes in the Windows L&F

### PR DESCRIPTION
**Pull request by mrserb**
Obsolete/unused classes are removed.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8260343](https://bugs.openjdk.java.net/browse/JDK-8260343): Delete obsolete classes in the Windows L&F

### Reviewers
 * [Alexey Ivanov](https://openjdk.java.net/census#aivanov) (@aivanov-jdk - **Reviewer**)
 * [Pankaj Bansal](https://openjdk.java.net/census#pbansal) (@pankaj-bansal - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/2216/head:pull/2216`
`$ git checkout pull/2216`

### CODE REVIEW

File: src/java.desktop/windows/classes/com/sun/java/swing/plaf/windows/WindowsButtonListener.java [hunk 1]
```diff
@@ -1,44 +0,0 @@
...
- * Button Listener.
- *
- * @author Rich Schiavi
- */
-public class WindowsButtonListener extends BasicButtonListener {
-    public WindowsButtonListener(AbstractButton b) {
-        super(b);
-    }
-    /*
-     This class is currently not used, but exists in case customers
```
Comment (pankaj-bansal) [inline src/java.desktop/windows/classes/com/sun/java/swing/plaf/windows/WindowsButtonListener.java:41]: Should not removing this public class need a CSR?

File: src/java.desktop/windows/classes/com/sun/java/swing/plaf/windows/WindowsComboBoxUI.java [hunk 2]
```diff
@@ -517,34 +517,6 @@ WindowsComboBoxUI getWindowsComboBoxUI() {
...
-
-    /**
-     * Subclassed to add Windows specific Key Bindings.
-     * This class is now obsolete and doesn't do anything.
-     * Only included for backwards API compatibility.
-     * Do not call or override.
-     *
-     * @deprecated As of Java 2 platform v1.4.
-     */
-    @Deprecated
```
Comment (prsadhuk) [inline src/java.desktop/windows/classes/com/sun/java/swing/plaf/windows/WindowsComboBoxUI.java:529]: Should we not mark it "forRemoval=true" before actually removing it?

File: src/java.desktop/windows/classes/com/sun/java/swing/plaf/windows/WindowsButtonListener.java (hunk 1 above)
Comment (mrserb) [reply src/java.desktop/windows/classes/com/sun/java/swing/plaf/windows/WindowsButtonListener.java:41]: it is not needed, this is not a public API.

File: src/java.desktop/windows/classes/com/sun/java/swing/plaf/windows/WindowsComboBoxUI.java (hunk 2 above)
Comment (mrserb) [reply src/java.desktop/windows/classes/com/sun/java/swing/plaf/windows/WindowsComboBoxUI.java:529]: it is not needed, this is not a public API.

**Review by aivanov-jdk — APPROVED** [review]

**Review by pankaj-bansal — APPROVED** [review]

**Pull request closed without merging**